main.go

```diff
@@ -126,6 +128,17 @@ func wrapMain() error {
 			Usage:  "Git tag",
 			EnvVar: "DRONE_TAG",
 		},
+		cli.StringSliceFlag{
+			Name:   "rollout_check",
```

Comment:
can this be called wait_deployments or something like that, to make it obvious that the values are Deployments?

Reply:
It's not necessary deployments, it could be also daemonsets or statefulsets.

Reply:
not the way you've set it up and documented it here

Reply:
I haven't tried, but I think this should work:
```
rollout_check: [gateway-server, deployment/gateway-server, daemonsets/some-set]
```

Reply:
that sounds great, but `deployment` is hardcoded in the command below (L439)

Reply:
Right, I see. I'll do it for `deployment` only for now.